systems/registry/node/pkg/db/model.go

```diff
@@ -151,3 +62,20 @@ func (s *Site) BeforeSave(tx *gorm.DB) (err error) {
 
 	return nil
 }
+
+func ParseNodeState(s string) ukama.NodeState {
```

Comment:
This parsing func should also be part of common i guess.

Reply:
addressed